api/src/queries.js

```diff
@@ -0,0 +1,24 @@
+const items = {
+  get: `{
+    query(func: has(code)) @filter(eq(code, $code)) {	
+      code
+      description
+    }
+  }`,
+  search: `{
```

Comment:
Same sort of thing as above
```suggestion
  search: `query vars($search: string) {
```